tasks/get_current_archive_list/get_current_archive_list.py

```diff
@@ -119,10 +121,20 @@ def task(
         # On error, set job status to failure.
         LOGGER.error(f"Encountered a fatal error: {fatal_exception}")
         # noinspection PyArgumentList
-        update_job_with_failure(job_id, str(fatal_exception), user_engine)
+        orca_shared.reconciliation.shared_reconciliation.update_job(
```

Comment:
See comments on possible signature changes to this function call.

Reply:
Fixed